scss/modules/_site-header.scss

```diff
@@ -84,6 +85,10 @@
   @include media($med) {
     display: inline-block;
   }
+
+  img {
```

Comment:
Minor thing, but we've been putting the `@include media()` mixins at the bottom, so the `img { }